src/components/common/ConfirmationWizard.js

```diff
@@ -131,13 +131,15 @@ class ConfirmationWizard extends React.Component {
         this.setState({
           errors: { password: 'Invalid password' }
         })
+        return undefined
```

Comment:
I like something like this more:

```js
      .then(isValid => {
        if (isValid) {
          this.submitWizard()
          return
        }
        this.setState({
          errors: { password: 'Invalid password' }
        })
      })
```

Reply:
Applied here https://github.com/autonomoussoftware/metronome-wallet-desktop/pull/484/commits/8ca46d4f535046a5eaeeaf2594a7d10893ae2585 . I went for that approach due to the eslint rule (because it was assuming `submitWizard` returned something) but given it does not, this works better!